src/training/_hooks.py

```diff
@@ -73,13 +73,17 @@ def after_run(self, context, values):
       throughput = self._steps_completed * float(self._batch_size) / float(duration)
 
       logging.info(LogSessionHook._MESSAGE_FORMAT,
-                   run_time, self._steps_completed, duration, throughput)
+                   self._steps_completed, run_time, duration, throughput)
 
 
 class LogTrainingHook(tf.train.SessionRunHook):
-  """Logs the training job by logging progress as well as producing summary events.
+  """Writes training loss into the logs, and writes summary events.
+
+  Two summary events are written
+  1) training loss of the current training step
+  2) running training throughput
   """
-  _MESSAGE_FORMAT = 'Global steps: %d; Duration: %d sec; Throughput: %.1f instances/sec; Loss: %.3f'
+  _MESSAGE_FORMAT = 'Global step %d: Training loss: %.3f'
```

Comment:
Similarly, not sure I understand this.

1. Why remove the throughput? And duration of the job?
2. Steps is generally plural everywhere in APIs.